src/main/ts/rules/NoDirectEditorEvents.ts

```diff
@@ -17,7 +17,7 @@ export const noDirectEditorEvents: Rule.RuleModule = {
     const filename = context.getFilename();
```

Comment:
Maybe just run the normalizeFilePath here instead then keep the endsWidth just name filename normalizedPath as you did at the other place.

Reply:
Yeah good point... were you thinking that we should not call `normalizePath` within each of the `PathUtils` functions then? I'd be a little worried someone might forget to call it beforehand if we did that though 🤷

Reply:
I ended up just deeming it an acceptable risk and it saves processing overhead to just do the normalize call once when getting the filename (not that it's a lot but it adds up). To help combat it a little I renamed the variables from `filePath` to `normalizedFilePath`.

Reply:
That is fine then all these are consistent. We could make the PathUtils a bit of a helper that does the normalization as well and has things to match on paths if we end up doing this alot.